core/templates/services/utils.service.ts

```diff
@@ -50,26 +48,24 @@ export class UtilsService {
     }
     return true;
   }
-
   // The function here is to check whether the argument is a string. So, we
   // cannot have any specific type defined for the argument and the argument
   // is given a generic type of Object.
   /**
-   * @param {Object} input - the object to be checked.
-   * @return {boolean} - true if input is string, false otherwise.
-   */
+  * @param {Object} input - the object to be checked.
```

Comment:
ANy expected reason for these changes?

Reply:
In the commit "_removing unwanted changes from core/templates/services/utils.service.ts_" I restored the spaces so that they no longer differ from the develop branch.